integration/rules/javascript_test.go

```diff
@@ -22,6 +22,11 @@ func TestJavascriptLangSessionDataflow(t *testing.T) {
 	runRulesTest("javascript/lang/session", "dataflow", "javascript_session", t)
 }
 
+func TestJavascriptWeakEncryption(t *testing.T) {
+	t.Parallel()
+	runRulesTest("javascript/lang/weak_encryption", "dataflow", "javascript_weak_encryption", t)
+}
```

Comment:
Any reason to not have dataflow *and* summary as we have for the others?

Reply:
Will add it, however I don't see the benefit of testing the same thing multiple times.
I believe it would be better to just keep the summary and remove dataflow test since summary utilises dataflow.